libr/core/cmd_search.c

```diff
@@ -2886,5 +2882,8 @@ static int cmd_search(void *data, const char *input) {
 		r_cons_newline ();
 	}
 	r_list_free (param.boundaries);
+	r_list_purge (core->search->kws);
```

Comment:
core shouldnt touch search internals, must use the api

Reply:
Added r_search_fini_one to destroy fields used by the current search.